### PR TITLE
Publish expired items and move item to text archive.

### PR DESCRIPTION
**Pull request by mdhaman**
- [x] Expire Published items.
- [x] Schedule the purge job for published items
- [x] Move published item to text archive.

### CODE REVIEW

**Comment by mdhaman** [comment]
@marwoodandrew I have item_id to text_archive as multiple version of the same items are required.

File: server/apps/prepopulate/app_initialize.py [hunk 1]
```diff
@@ -18,7 +18,10 @@
     'stages': ('stages.json', 'desk'),
     'groups': ('groups.json', ''),
     'vocabularies': ('vocabularies.json', ''),
-    'content_templates': ('content_templates.json', 'template_name')
+    'content_templates': ('content_templates.json', 'template_name'),
+    'published': (None, [('item_id', pymongo.ASCENDING),
+                         ('state', pymongo.ASCENDING)]),
+    'published': (None, '_created')
```
Comment (amagdas) [inline server/apps/prepopulate/app_initialize.py]: This should be joined in the pair above.

File: server/apps/publish/commands.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                            format(item['_id'], item['headline'], item['expiry'], now))
+                superdesk.get_resource_service('published').delete_action({'_id': str(item['_id'])})
+            items = self.get_expired_items(now)
+
+    def get_expired_items(self, now):
+        logger.info('Get expired content from published')
+        query_filter = self.get_query_for_expired_items(now)
+        req = ParsedRequest()
+        req.sort = '_created'
+        req.max_results = 1
```
Comment (amagdas) [inline server/apps/publish/commands.py]: Is this on purpose, to limit to one result... ?

Comment (mdhaman) [inline server/apps/publish/commands.py]: its typo. will correct that...

**Comment by amagdas** [comment]
:+1:

File: server/apps/prepopulate/app_initialize.py (hunk 1 above)
Comment (mdhaman) [inline server/apps/prepopulate/app_initialize.py]: yes right. 
I have removed _created index.

**Comment by akintolga** [comment]
:+1:

**Pull request merged**